### PR TITLE
Set map canvas extent to one obtained from the model (t_ili2db_column_prop table)

### LINKED ISSUE #83: Map extent from INTERLIS models
Since INTERLIS models have coordinate ranges that play a role when validating data (i.e., avoiding data export when features are outside of it), couldn't we read the allowed extent and do things like the following?

 + Set initial map extent 
 + Create a spatial bookmark called "Allowed extent" (and open the Spatial Boormarks Panel each time a project is generated)
 + Show a warning message when saving features out of the allowed extent

### PR DESCRIPTION
**Pull request by gacarrillor**
We pick the first layer extent for setting the canvas extent. We could have used the union of all extents in the `t_ili2db_column_prop` table, but we didn't since the base geometry model is almost set in stone and all layers use the same definitions from such model. 

@m-kuhn What do you think about such approach? Do Swiss models use several base geometry models or extent definitions? 

2 tests added.

Fix #83

### CODE REVIEW

File: projectgenerator/libqgsprojectgen/dbconnector/db_connector.py [hunk 1]
```diff
@@ -58,6 +58,7 @@ def get_tables_info(self):
                 type  (Geometry Type)
                 kind_settings
                 ili_name
+                extent (insert coordinates xmin, ymin, xmax, ymax)
```
Comment (m-kuhn) [inline projectgenerator/libqgsprojectgen/dbconnector/db_connector.py]: needs to be semicolon, not comma separated, reading the code?

**Comment by m-kuhn** [comment]
I like it :+1:

File: projectgenerator/libqgsprojectgen/dbconnector/db_connector.py (hunk 1 above)
Comment (gacarrillor) [reply projectgenerator/libqgsprojectgen/dbconnector/db_connector.py]: Right, done.

**Comment by gacarrillor** [comment]
Just added two tests for this. It should be ready now.

**Pull request merged**